### PR TITLE
Update PR & Issue templates

### PR DESCRIPTION
**Pull request by thomasheremans**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- 🎉🚀 Thanks for submitting a PR!
```
Comment (dB2510) [inline .github/pull_request_template.md:1]: Seems like this template is copied from charon, I think it won't applicable everywhere else. the reason being we don't push main features in this repo.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
-To help us tend to your issue faster, please search our currently open issues before submitting a new one.
-Existing issues often contain information about workarounds, resolution, or progress updates.
+<!--Describe in detail the problem to be solved by this feature/enhancement and why it is important. -->
 
--->
+# 🛠️ Proposed solution
 
-# 🚀 Feature Request
+- [ ] Approved design doc: *link*
+- [ ] Core team consensus on the proposed solution
```
Comment (dB2510) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: Not needed in cdvn repo

**Review by dB2510 — CHANGES_REQUESTED** [review]
In my mind cdvn and cdvc repos are very much user facing repositories. These are used as templates to run DVs, so I don't think it is justified to copy charon specific issue and PR templates to every repo. Can we keep it simple here?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (thomasheremans) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Thanks, removed from the other repos as well

File: .github/pull_request_template.md (hunk 1 above)
Comment (thomasheremans) [reply .github/pull_request_template.md:1]: I think some structure to any PR is useful, I tried to remove any non-relevant item but let me know if there are any other that don't make sense

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+📐 Format this PR template as follows:
+   - 1️⃣ PR title (above): concise overview
+   - 2️⃣ PR body: Replace 'body...' with detailed description of the change.
+   - 3️⃣ category: Pick one, delete the rest.
+   - 4️⃣ ticket: Replace #000 with link to a GitHub issue (or 'none' if PR is trivial).
+🧑‍🎓 Please review our contribution guide https://github.com/ObolNetwork/charon/blob/main/docs/contributing.md
```
[the remark this answers is not in the document]
Comment (thomasheremans) [reply .github/pull_request_template.md:7]: Doesn't it make sense for this repo as well? Or maybe we should standardise the contribution guide?

[the remark this answers is not in the document]
Comment (dB2510) [reply .github/pull_request_template.md:7]: > Doesn't it make sense for this repo as well?

yeah that contribution guide is written only charon in mind. 

> Or maybe we should standardise the contribution guide?

+1 for standard contribution guide. Also cdvn and cdvc repos are just for user deployments and not for feature development, so it doesn't make sense to have charon's contribution guide here.

**Review by dB2510 — APPROVED** [review]

**Comment by dB2510** [comment]
@thomasheremans I have removed pull request template for now. Will add if we need it. Merging this PR for now.

**Pull request merged**